velox/functions/prestosql/tests/ArrayHasDuplicatesTest.cpp

```diff
@@ -143,5 +143,5 @@ TEST_F(ArrayHasDuplicatesTest, nullFreeStrings) {
 
       });
   auto expected = makeFlatVector<bool>({false, true, false, true});
-  testExpr(expected, "array_has_duplicates(C0)", {array});
+  testExpr(expected, "array_has_dupes(C0)", {array});
```

Comment:
Rather than changing what I think is an arbitrary test to use the new alias, maybe add a helper function that runs the test with both aliases, and use that in all the tests.

That way if this test is removed or modified we don't lose coverage for the new alias.

Same applies to the other test file.